### PR TITLE
howto/rootless: Add setcap alternative

### PR DESCRIPTION
**Pull request by cmouse**

### CODE REVIEW

File: source/configuration_manual/howto/rootless.rst [hunk 1]
```diff
@@ -48,6 +48,25 @@ example configuration file exists in
...
+without losing chroot and privileged ports.
+
+Use following commands to enable this
+
+.. code:: bash
+
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/submission-login
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/pop3-login
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/imap-login
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/lmtp
```
Comment (sirainen) [inline source/configuration_manual/howto/rootless.rst:65]: But binding is done by dovecot master process, shouldn't it be the one getting cap_net_bind_service?

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/configuration_manual/howto/rootless.rst (hunk 1 above)
Comment (cmouse) [reply source/configuration_manual/howto/rootless.rst:65]: i tested that this works.

Comment (sirainen) [reply source/configuration_manual/howto/rootless.rst:65]: I don't understand why though. Does it then also fail if cap_net_bind_service is not set for these?

Comment (cmouse) [reply source/configuration_manual/howto/rootless.rst]: Fixed.

**Review by sirainen — APPROVED** [review]

**Pull request merged**